SortWoSTL/namesSortWoSTL.h

```diff
@@ -0,0 +1,26 @@
+#ifndef NAMESSORTWOSTL_H
+#define NAMESSORTWOSTL_H
+#include <iostream>
+#include <fstream>
+#include <string>
+
+class NamesSortWoSTL{
+private:
+    static int const size = 5000;
```

Comment:
please define your order)) - it's either "static const int" or "static int const" (1st is more common)